interface/main/finder/dynamic_finder.php

```diff
@@ -37,10 +37,12 @@
 $header0 = "";
 $header = "";
 $coljson = "";
-$res = sqlStatement("SELECT option_id, title FROM list_options WHERE " .
+$orderjson = "";
+$res = sqlStatement("SELECT option_id, title, toggle_setting_1 FROM list_options WHERE " .
     "list_id = 'ptlistcols' AND activity = 1 ORDER BY seq, title");
 while ($row = sqlFetchArray($res)) {
     $colname = $row['option_id'];
+    $colorder = generate_display_field(array('data_type'=>'1','list_id'=>'Sort_Direction'), $row['toggle_setting_1']);
```

Comment:
Been looking into this. do note that this will translate what is returned, so it could get change to something else :)
Will be better to grab the title directly via a sql query from list_options.

Reply:
Fixed